cognee/tasks/schema/ingest_database_schema.py

```
@@ -0,0 +1,113 @@
+from cognee.infrastructure.databases.relational.get_migration_relational_engine import (
+    get_migration_relational_engine,
+)
+from sqlalchemy import text
+from cognee.tasks.schema.models import DatabaseSchema, SchemaTable, SchemaRelationship
+from cognee.infrastructure.databases.relational.create_relational_engine import (
+    create_relational_engine,
+)
+from datetime import datetime
+
+
+async def ingest_database_schema(
```

Pylint (3.3.7)

[refactor] Too many local variables (24/15)

(R0914)